### DISCUSSION TITLE
New monster - Latcher's Cub

### DISCUSSION THREAD

**Discussion by Gobrich** [FakeFishGames/Barotrauma#10447] [Ideas and suggestions]
I suggest adding the cubs of the Latcher to the game, which would live only in caves. The cubs should have the same functionality as the creature from the Abyss. I imagine them as very dangerous enemies capable of destroying an unprepared expedition. The new monster should not have glowing eyes like the parent. This will make players pay more attention to the environment, use warning lights and flashlights, which fits well with the additions from the unstable version 0.20

**Comment by ZOVfurry** [comment]
зачем
